static/js/public/fullRecord.js

```diff
@@ -149,15 +149,31 @@ define('fullRecord', ['module', 'jquery', 'JP2Viewer', 'StructureView', 'dataTab
 
 		$childFilesTable.DataTable({
 			ajax: {
-				processing: true,
 				url: '/listJson/' + $childFilesTable.attr('data-pid') + "?rows=2000",
 				dataSrc: function(d) {
 					return d.metadata;
-				}
+				},
+				data: function (d) {
+					const sorts = ['title', 'fileFormatDescription', 'fileSize'];
+					const sortOrder = {'asc': 'normal', 'desc': 'reverse'};
+					d.anywhere=d.search['value'];
+					d.length=10;
+					d.rollup=false;
+					d.sort=sorts[d.order[0]['column'] - 1] + ',' + sortOrder[d.order[0]['dir']];
+				},
+				dataFilter: function(data){
+					let json = jQuery.parseJSON( data );
```

Comment:
Extra spaces around "data" and jQuery.parseJson is deprecated in jQuery. JSON.parse() is recommended now.